### ISSUE TITLE
Github should add a negative star button just for skids like you 👍

### ISSUE THREAD

**Issue by ar1ja** [Freaked-Dude/Freak#5]

**Comment by Airaharune** [comment]
Who are you

**Comment by eternalodball** [comment]
> _No description provided._

Totally agree with you from where these kids there account should be banned

**Comment by ar1ja** [comment]
> > _No description provided._
> 
> Totally agree with you from where these kids there account should be banned

Your account is not much better

**Comment by eternalodball** [comment]
> > > _No description provided._
> > 
> > 
> > Totally agree with you from where these kids there account should be banned
> 
> Your account is not much better

actually i don't care , coding is my hobby not job i don't like this guy as he did stuff that aren't required . i will advice you one thing instead of advising others look into yourself how arrogant you are . Thanks and have a good day

**Comment by eternalodball** [comment]
> > > _No description provided._
> > 
> > 
> > Totally agree with you from where these kids there account should be banned
> 
> Your account is not much better

and by the way i write codes to be understood by others and i think i have far better knowledge than you on python idk about c++ so go and learn first bye !!

**Comment by ar1ja** [comment]
> > > > _No description provided._
> > > 
> > > 
> > > Totally agree with you from where these kids there account should be banned
> > 
> > 
> > Your account is not much better
> 
> and by the way i write codes to be understood by others and i think i have far better knowledge than you on python idk about other languages so go and learn first bye , adding some random shit on your github profile won't make you devloper kiddo xd

I've been literally doing python for over 5 years lmfao

**Comment by ar1ja** [comment]
> > > > _No description provided._
> > > 
> > > 
> > > Totally agree with you from where these kids there account should be banned
> > 
> > 
> > Your account is not much better
> 
> actually i don't care , coding is my hobby not job i don't like this guy as he did stuff that aren't required . i will advice you one thing instead of advising others look into yourself how arrogant you are . Thanks and have a good day

And your "instagram hacker", "instagram spammer" and stuff like that will never get you a job <3

**Comment by ar1ja** [comment]
And do you really think [this code](https://github.com/eternalodball/inscreate/blob/main/ac.py) is better and more readable than [this code](https://github.com/TruncatedDinosour/blog.ari-web.xyz/blob/main/scripts/blog) for example, even though that script is defo not my prettiest creation, it's much better than whatever anti-PEP8 script kiddie code you got there @eternalodball

**Comment by ar1ja** [comment]
Even in your [small scripts](https://github.com/eternalodball/insbrute/blob/main/main.py) your code is not the best, at least follow PEP8... like use flake8 and stuff, I actually have [my vim](https://ari-web.xyz/dotfile/gentoo/dotfiles/editors/vim) set up for good development, esspecially python with formatting and stuff, check it out @eternalodball

**Comment by eternalodball** [comment]
> Even in your [small scripts](https://github.com/eternalodball/insbrute/blob/main/main.py) your code is not the best, at least follow PEP8... like use flake8 and stuff, I actually have [my vim](https://ari-web.xyz/dotfile/gentoo/dotfiles/editors/vim) set up for good development, esspecially python with formatting and stuff, check it out @eternalodball

lol okay my time is not worth wasting on such a arrogant ppl like you

**Comment by ar1ja** [comment]
> > Even in your [small scripts](https://github.com/eternalodball/insbrute/blob/main/main.py) your code is not the best, at least follow PEP8... like use flake8 and stuff, I actually have [my vim](https://ari-web.xyz/dotfile/gentoo/dotfiles/editors/vim) set up for good development, esspecially python with formatting and stuff, check it out @eternalodball
> 
> lol okay my time is not worth wasting on such a arrogant ppl like you

Cool, gonna start correcting my grammar too soon? lolllllllllllllllllllllll

**Comment by eternalodball** [comment]
> > > Even in your [small scripts](https://github.com/eternalodball/insbrute/blob/main/main.py) your code is not the best, at least follow PEP8... like use flake8 and stuff, I actually have [my vim](https://ari-web.xyz/dotfile/gentoo/dotfiles/editors/vim) set up for good development, esspecially python with formatting and stuff, check it out @eternalodball
> > 
> > 
> > lol okay my time is not worth wasting on such a arrogant ppl like you
> 
> Cool, gonna start correcting my grammar too soon? lolllllllllllllllllllllll

good luck on it

**Comment by eternalodball** [comment]
i actually didn't focus much on your replies and replied blindly but now after reading it i realised you are not only arrogant but possessive about yourself ppl like you can't do anything in their life except for proving themselves that i am right one day you will fall and when you fall it will be a depression for you . Bye probably last reply ;0 

and yeah i agree i am not better than you in coding just for you sympathy so you won't shout again

**Comment by ar1ja** [comment]
> i actually didn't focus much on your replies and replied blindly but now after reading it i realised you are not only arrogant but possessive about yourself ppl like you can't do anything in their life except for proving themselves that i am right one day you will fall and when you fall it will be a depression for you . Bye probably last reply ;0
> 
> and yeah i agree i am not better than you in coding just for you sympathy so you won't shout again

wtf i never shouted lol, how am I possesive about myself, you're the one who called me a skid and I tried to disprove it

**Comment by Anon0369** [comment]
I agree the only reason he said that is because u put up a simple script on an open source platform and then refuse to tell the user and password. U should at least say that it requires them before people install it. Then they don't have to do that for nothing and then remove it. Waste of time. U van guess the username pretty East but the pass is what? Freaked-Dude , lowercase? What?

**Comment by Anon0369** [comment]
Username is Freak Password is Login. You have to read the File to find it

**Comment by ar1ja** [comment]
> Username is Freak Password is Login. You have to read the File to find it

?
